src/main/java/com/adaptavist/tm4j/jenkins/extensions/CustomTestCycle.java

```diff
@@ -2,18 +2,15 @@
 
 import static org.apache.commons.lang.StringUtils.isBlank;
 
-import com.adaptavist.tm4j.jenkins.utils.GsonUtils;
-import com.google.gson.reflect.TypeToken;
-import java.util.HashMap;
-import java.util.Map;
+import hudson.EnvVars;
 import org.kohsuke.stapler.DataBoundConstructor;
 
 public class CustomTestCycle {
```

Comment:
This class now will work only as a holder to hold information to/from the UI. All validations are now moved to `ExpandedCustomTestCycle`.